.github/workflows/wheels.yml

```diff
@@ -121,10 +183,29 @@ jobs:
         run: |
           python -m pip install ./artifact/mrob-*-macosx_10_9_x86_64.whl numpy pytest
 
-      - name: "Python unit tests"
+      - name: Install mrob & numpy on Windows 64bit
+        if: ${{ matrix.os == 'windows-latest' }}
+        run: |
+          python -m pip install .\artifact\$(dir artifact -Name | findstr mrob-.*-win_amd64.whl) numpy pytest
+
+      - name: Install mrob & numpy on Windows 32bit
+        if: ${{ matrix.os == 'windows-2019' }}
+        run: |
+          python -m pip install .\artifact\$(dir artifact -Name | findstr mrob-.*-win32.whl) numpy
+
+      - name: Python unit tests
+        if: ${{ matrix.os != 'windows-2019' }}
         run: |
           python -m pytest mrobpy
 
+      - name: Python unit tests
+        if: ${{ matrix.os == 'windows-2019' }}
+        run: |
+          $files = Get-ChildItem -Path python_examples -Include *.py
+          foreach ($f in $files){
+            python $f.FullName || EXIT /B !ERRORLEVEL!
+          }
+
```

Comment:
`python -m pytest mrobpy`  causes a `Windows fatal exception: access violation` on some versions of Python (on  Windows, x86)